service/user.ts

```diff
@@ -14,6 +14,7 @@ import {
 } from "proto/api_pb";
 import { AuthReq, CompleteTokenLoginReq } from "proto/auth_pb";
 import client from "service/client";
+import newClient from "service/newClient";
```

Comment:
is there a way to avoid calling this file "newClient"? Maybe "restClient" or some other attribute to differentiate it?